src/sentry/models/auditlogentry.py

```diff
@@ -62,6 +62,12 @@ class AuditLogEntryEvent(object):
     RULE_EDIT = 81
     RULE_REMOVE = 82
 
+    SERVICEHOOK_ADD = 90
```

Comment:
Don't these also need to be added into `AuditLogEntry.event` choices? As well as `AuditLogEntry.get_note`

Reply:
derp, been so long since ive built anything I forget how it works